libs/core/src/lib/utils/pipes/dateFormat.pipes.ts

```diff
@@ -0,0 +1,48 @@
+import { Inject, Optional, Pipe, PipeTransform } from '@angular/core';
+import { DATE_TIME_FORMATS, DateTimeFormats } from '../../datetime/datetime-formats';
+import { DatetimeAdapter } from '../../datetime/datetime-adapter';
+
+@Pipe({
+    name: 'dateFormatPipe'
+})
+export class DateFormatPipe<D> implements PipeTransform {
+    constructor(
+        private _dateTimeAdapter: DatetimeAdapter<D>,
+        @Optional() @Inject(DATE_TIME_FORMATS) private _dateTimeFormats: DateTimeFormats
+    ) {}
+
+    transform(date: D): any {
```

Comment:
should we change any to string?
as I see `this._dateTimeAdapter.format` returns `string`